test/SQONBuilder.spec.ts

```diff
@@ -114,7 +114,7 @@ describe('SQONBuilder', () => {
 			expect(emptySQON()).deep.equal(expected);
 		});
 	});
-	describe('Static functions', () => {
+	describe('static functions', () => {
```

Comment:
on this point, I've found numbering tests to be highly useful.
this would be "7. Static..."

Reply:
I would hate maintaining this order as things are added/removed/rearranged. Would be happy to have the output decorated this way.